docs/_docs/swarm/run.md

```diff
@@ -0,0 +1,140 @@
+---
+nav_order: 2
+redirect_from: /docs/swarm/run.html
+---
+
+# Run
+
+***
+
+The run command of the HiveMQ Swarm command line offers a set of commands to start and stop runs.
+
+```
+mqtt swarm run
+```
+
+***
+
+## Commands
+
+| Command | Explanation             |
+|---------|-------------------------|
+| start   | See [start](#start-run) |
+| stop    | See [stop](#stop-run)   |
+
+***
+
+## Options
+
+| Option | Long Version | Explanation                                | Default                 |
+|--------|--------------|--------------------------------------------|-------------------------|
+| `-url` |              | The URL of the HiveMQ Swarm Rest endpoint. | `http://localhost:8888` |
+| `-r`   | `--run-id`   | The id of the run to stop.                 | `The current run`       |
+
+### Logging Options
+
+{% include options/logging-options.md %}
+
+### Help Options
+
+{% include options/help-options.md defaultHelp=true %}
+
+***
+
+# Start Run
+
+***
+
+Start a scenario, either blocking or detached.
+
+```
+mqtt swarm run
+```
+
+***
+
+## Options
+
+| Option | Long Version | Explanation                                           | Default                 |
+|--------|--------------|-------------------------------------------------------|-------------------------|
+| `-url` |              | The URL of the HiveMQ Swarm Rest endpoint.            | `http://localhost:8888` |
+| `-f`   | `--file`     | The scenario to execute.                              | required                |
+| `-r`   | `--run-id`   | The id of the run to stop.                            | `The current run`       |
```

Comment:
```suggestion
| `-r`   | `--run-id`   | The id of the run to start.                            | `The current run`       |
```
I don't see this option in the CLI, can you actually name a run on start?

Reply:
Will also remove that row.